tests/pagila-standby/copydb.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+set -x
+set -e
+
+# This script expects the following environment variables to be set:
+#
+#  - PGCOPYDB_SOURCE_PGURI
+#  - PGCOPYDB_SOURCE_STANDBY_PGURI
+#  - PGCOPYDB_TARGET_PGURI
+#  - PGCOPYDB_TABLE_JOBS
+#  - PGCOPYDB_INDEX_JOBS
+
+# make sure source and target databases are ready
+pgcopydb ping
+
+# sleep 5 seconds to make sure standby is ready
+sleep 5
+
+grep -v "OWNER TO postgres" /usr/src/pagila/pagila-schema.sql > /tmp/pagila-schema.sql
+
+psql -o /tmp/s.out -d ${PGCOPYDB_SOURCE_PGURI} -1 -f /tmp/pagila-schema.sql
+psql -o /tmp/d.out -d ${PGCOPYDB_SOURCE_PGURI} -1 -f /usr/src/pagila/pagila-data.sql
+
+pgcopydb clone --skip-ext-comments --notice \
+         --source ${PGCOPYDB_SOURCE_STANDBY_PGURI} \
+         --target ${PGCOPYDB_TARGET_PGURI}
+
```

Comment:
Can we test CDC(i.e. --follow) as well?

Reply:
Not yet, because that requires Postgres 16. Let's first focus on the clone operations, and when we have that in we will be in a great position to see about supporting follow operations too.